overlord/restart/restart.go

```diff
@@ -321,6 +332,10 @@ func FinishTaskWithRestart(task *state.Task, status state.Status, rt RestartType
 				if err := notifyRebootRequiredClassic(snapName); err != nil {
```

Comment:
Do we still need to the `update-notifier` script, can't we just use the `userd` approach for both classic and core?

Reply:
Once the UI is solid, that might be worth considering. For now, it is probably best to keep this Core Desktop only, since we don't have update-notifier there, and likely can't due to confinement.